region/brinstar/blue.json

```diff
@@ -135,38 +131,84 @@
           "from": 1,
           "to": [
             {
-              "id": 5,
+              "id": 6,
               "requires": [
-                "Morph",
-                {"obstacle":{
-                  "id": "A",
-                  "requires": [
-                    {"or":[
-                      "Bombs",
-                      {"and":[
-                        "PowerBomb",
+                {"or":[
+                  {"cost":[
+                    {"cost":[
+                      {"enemyDamage": {
+                        "enemy": "Sidehopper",
+                        "type": "contact",
+                        "hits": 1
+                      }}
+                    ]}
+                  ]},
+                  {"obstacle":{
+                    "id": "C",
+                    "requires": [
+                      {"or":[
                         {"cost":[
-                          {"ammo": [
-                            {
-                              "type": "PowerBomb",
-                              "count": 1
-                            }
+                          {"enemyDamage": {
+                            "enemy": "Sidehopper",
+                            "type": "contact",
+                            "hits": 8
+                          }}
+                        ]},
+                        {"and":[
+                          {"cost":[
+                            {"enemyDamage": {
+                              "enemy": "Sidehopper",
+                              "type": "contact",
+                              "hits": 1
+                            }}
+                          ]},
+                          {"or":[
+                            {"enemyKill":{
+                              "enemies": [
+                                [ "Sidehopper", "Sidehopper", "Sidehopper"]
+                              ],
+                              "explicitWeapons": ["Missile", "Super"]
+                            }},
+                            "ScrewAttack",
+                            "Plasma"
+                          ]}
+                        ]},
+                        {"and":[
+                          {"cost":[
+                            {"enemyDamage": {
+                              "enemy": "Sidehopper",
+                              "type": "contact",
+                              "hits": 2
```

Comment:
I think it's perfectly fair to expect getting hit only once when entering the room here and then morph down into the corner below while the PB is going off.

Reply:
And you can do the ol' morph ball juke with the other guy. Sold.